ros/src/twist_controller/dbw_node.py

```diff
@@ -72,6 +83,11 @@ def loop(self):
             #                                                     <any other argument you need>)
             # if <dbw is enabled>:
             #   self.publish(throttle, brake, steer)
+
+
+            self.publish(0.9,0,0)
```

Comment:
could u add some comments here, something like `TODO: here is just some example to set throttle constant`